Team-C/test_wizard/test/widget_test.dart

```diff
@@ -5,26 +5,26 @@
 // gestures. You can also use WidgetTester to find child widgets in the widget
 // tree, read text, and verify that the values of widget properties are correct.
 
```

Comment:
I need help fixing these tests.
I've attached a screenshot of what running the app will look like, `flutter run` and it goes to the login screen but it can't seem to find the buttons nor text 
<img width="1198" alt="Screenshot 2024-06-29 at 4 40 45 PM" src="https://github.com/umgc/Summer2024/assets/155414703/e89f433d-4df4-4cb9-b462-347fa1586fed">
fields.  Any help is appreciated

Reply:
The user login has change to OAuth. We are using Moodle to authenticate so the login should direct users to Moodle to authenticate.

Reply:
the `authenticate` method in  [auth_service.dart](https://github.com/umgc/Summer2024/pull/407/files#diff-eca7af2844275f241cdd5cdf7ed6d9b0cabebeaf1aace677c400eeb9c01864bb) under the hood authenticates to a moodle endpoint using moodle credentials, after-which it will redirect to the home screen of our test wizard application.